build.zig

```diff
@@ -1237,19 +1240,23 @@ fn build_go_client(
         }) catch unreachable;
         const resolved_target = b.resolveTargetQuery(query);
 
-        const lib = b.addStaticLibrary(.{
-            .name = "tb_client",
+        const root_module = b.createModule(.{
             .root_source_file = b.path("src/tigerbeetle/libtb_client.zig"),
             .target = resolved_target,
             .optimize = options.mode,
+            .stack_protector = false,
```

Comment:
> Is stack protector removal intentional?

Stack protector was moved to the `root_module` declaration.

> Have no idea what it does, but sounds like it might be important?

IIRC, it caused issues with CGO, so we disabled it only for the go client.

Reply:
Ah, see it now, thanks!